CHANGELOG.md

```diff
@@ -1,49 +1,229 @@
+  [#14448](https://github.com/rust-lang/cargo/pull/14448)
+- Improved error messages for missing features.
   [#14436](https://github.com/rust-lang/cargo/pull/14436)
-- Enhanced the dependency update status message, including
-  - Reports incompatible packages with precise Rust version
-    [#14457](https://github.com/rust-lang/cargo/pull/14457)
-  - Reports incompatible-with-rustc when MSRV-resolver is disabled
-    [#14459](https://github.com/rust-lang/cargo/pull/14459)
-  - Reports compatible, incompatible, and direct-dep updates with different
-    colors and messages.
-    [#14461](https://github.com/rust-lang/cargo/pull/14461)
-  - The `Locking` status message stops showing workspace members.
-    [#14445](https://github.com/rust-lang/cargo/pull/14445)
 - Log details of `rustc` invocation failure if no errors are seen
   [#14453](https://github.com/rust-lang/cargo/pull/14453)
-- cargo-package: Don't automatically include the current crate when packaging
-  [#14488](https://github.com/rust-lang/cargo/pull/14488)
+- Uplifted `windows-gnullvm` import libraries, aligning them with `windows-gnu`.
+  [#14451](https://github.com/rust-lang/cargo/pull/14451)
+- Suggest `cargo info` command in the `cargo search` result
+  [#14537](https://github.com/rust-lang/cargo/pull/14537)
+- Improved error reporting when a feature is not found in `activated_features`.
+  [#14647](https://github.com/rust-lang/cargo/pull/14647)
+- Enhanced dependency update status messages, now displaying updates
+  (compatible, incompatible, direct-dep) in different colors,
+  along with messages and MSRVs.
+  [#14440](https://github.com/rust-lang/cargo/pull/14440)
+  [#14457](https://github.com/rust-lang/cargo/pull/14457)
+  [#14459](https://github.com/rust-lang/cargo/pull/14459)
+  [#14461](https://github.com/rust-lang/cargo/pull/14461)
+  [#14471](https://github.com/rust-lang/cargo/pull/14471)
+  [#14568](https://github.com/rust-lang/cargo/pull/14568)
+- The `Locking` status message no longer displays workspace members.
+  [#14445](https://github.com/rust-lang/cargo/pull/14445)
 
 ### Fixed
 
-- cargo-add: Perform fuzzy search when translating package names
+- Prevented duplicate library search environment variables when calling `cargo` recursively.
+  [#14464](https://github.com/rust-lang/cargo/pull/14464)
+- Don't double-warn about `$CARGO_HOME/config` not having `.toml` extension.
+  [#14579](https://github.com/rust-lang/cargo/pull/14579)
+- Correct diagnostic count message when using `--message-format json`.
+  [#14598](https://github.com/rust-lang/cargo/pull/14598)
+- cargo-add: Perform fuzzy searches when translating package names
   [#13765](https://github.com/rust-lang/cargo/pull/13765)
+- cargo-new: only auto-add new packages to the workspace relative to the manifest,
+  rather than the current directory.
+  [#14505](https://github.com/rust-lang/cargo/pull/14505)
+- cargo-package: Don't automatically include the current crate when packaging
```

Comment:
"include" can also be read as "in the tarball". I was confused by this.

Reply:
```
- ❗️ cargo-package: When using the `--package` flag, only the specified
  packages are packaged. Previously, the package in the current working
  directory was automatically selected for packaging.
```


After re-read the PR I think it justifies to have a compatibility note.

Moved to `Changed` with a big ❗️.

BTW, I didn't mention `--manifest-path` at all, although it is the package from either cwd or `--manifest-path` would be auto selected. That goes too detailed and I don't feel worthy.